packages/frontend-web/src/app/platform/websocketService.ts

```diff
@@ -81,6 +81,18 @@ export interface IPerUserData {
 //       When this is availabe, replace the "any" types in the code below.
 // TODO: API sending number IDs, but we expect strings due to the way the old REST code works.
 //       Convert for now.  But at some point need to refactor to use numbers.
+
+function fixObject(o: any) {
```

Comment:
Seems pretty clear that `o` is not any, but `{ categoryId: string | Stringable; id: string | Stringable, tagId: string |Stringable }`. where Stringable is something like `{ toString() : string }`

When you introduce new (especially small_) functions, much better to give them types.